stable_plugins/classical_ml/scikit_ml/optics-clustering/templates/optics-clustering_template.html

```diff
@@ -0,0 +1,102 @@
+{% extends "simple_template.html" %}
+
+{% block script %}
+<script>
+    var algorithm_enum_val = document.getElementById("algorithm_enum");
+    var metric_enum_val = document.getElementById("metric_enum");
+    var method_enum_val = document.getElementById("method_enum");
+    var epsilon_val = document.getElementById("epsilon");
+    var xi_val = document.getElementById("xi");
+    var min_cluster_size_val = document.getElementById("min_cluster_size");
+    var leaf_size_val = document.getElementById("leaf_size");
+    var minkowski_p_val = document.getElementById("minkowski_p");
+
+
+    var epsilon_dis = epsilon_val.parentNode.parentNode;
+    var xi_dis = xi_val.parentNode.parentNode;
+    var predecessor_correction_dis = document.getElementById("predecessor_correction").parentNode.parentNode;
+    var min_cluster_size_dis = min_cluster_size_val.parentNode.parentNode;
+    var leaf_size_dis = leaf_size_val.parentNode.parentNode;
+    var minkowski_p_dis = minkowski_p_val.parentNode.parentNode;
+
+    function set_union(setA, setB) {
+        const _union = new Set(setA);
+        setB.forEach(elem => _union.add(elem));
+        return _union;
+    }
+
+    //seuclidean, wminkowski, mahalanobis need more params
+    const ball_tree_metrics = new Set(["braycurtis", "canberra", "chebyshev", "cityblock", "dice", "euclidean", "hamming", "haversine", "jaccard", "kulsinski", "l1", "l2", "mahalanobis", "manhattan", "matching", "minkowski", "rogerstanimoto", "russellrao", "seuclidean", "sokalmichener", "sokalsneath", "wminkowski"]);
+    const kd_tree_metrics = new Set(['chebyshev', 'cityblock', 'euclidean', 'l1', 'l2', 'manhattan', 'minkowski']);
+    const brute_metrics = new Set(['braycurtis', 'canberra', 'chebyshev', 'cityblock', 'correlation', 'cosine', 'dice', 'euclidean', 'hamming', 'haversine', 'jaccard', 'kulsinski', 'l1', 'l2', 'mahalanobis', 'manhattan', 'matching', 'minkowski', 'nan_euclidean', 'precomputed', 'rogerstanimoto', 'russellrao', 'seuclidean', 'sokalmichener', 'sokalsneath', 'sqeuclidean', 'yule']);
+    const auto_metrics = set_union(set_union(ball_tree_metrics, kd_tree_metrics), brute_metrics);
```

Comment:
see above

Reply:
I will not replace this function, according to @buehlefs wishes (https://github.com/UST-QuAntiL/qhana-plugin-runner/pull/98#discussion_r1395454774).